### ISSUE TITLE
muparser 2.2.6 version is no picked up correclty

### ISSUE THREAD

**Issue by davydden** [dealii/dealii#7413]
```
-- Found MUPARSER_INCLUDE_DIR
--   MUPARSER_VERSION: 2.2.5
--   MUPARSER_LIBRARIES: /Users/davydden/spack/opt/spack/darwin-mojave-x86_64/clang-10.0.0-apple/muparser-2.2.6-wj7ahumgn3m744mwnc23ck6kkrask45i/lib/libmuparser.dylib
--   MUPARSER_INCLUDE_DIRS: /Users/davydden/spack/opt/spack/darwin-mojave-x86_64/clang-10.0.0-apple/muparser-2.2.6-wj7ahumgn3m744mwnc23ck6kkrask45i/include
-- Found MUPARSER
```

I have not checked whether it's `muparser` issue or `deal.II`, but put it here for now...

**Comment by masterleinad** [comment]
I can't reproduce
```
DEAL_II_WITH_MUPARSER set up with external dependencies
 #            MUPARSER_VERSION = 2.2.6
 #            MUPARSER_DIR = /mnt/data/darndt/muparser-2.6
 #            MUPARSER_INCLUDE_DIRS = /mnt/data/darndt/muparser-2.6/include
 #            MUPARSER_LIBRARIES = /mnt/data/darndt/muparser-2.6/lib/libmuparser.so
```
with https://github.com/beltoforion/muparser/releases/tag/v2.2.6.1.
Are you sure you are running from a clean build directory?

**Comment by davydden** [comment]
> Are you sure you are running from a clean build directory?

yes, but I install `2.2.6`, not `2.2.6.1`, perhaps there were some fixes...

**Comment by davydden** [comment]
yes, `2.2.6.1` is picked up correctly from `deal.II`, so it must be issues with `muparser`.

**Comment by davydden** [comment]
yes, `2.2.6` contains

```
#define MUP_VERSION _T("2.2.5")
```

**Comment by tamiko** [comment]
@davydden Jup. The include file in the 2.2.6 tarball contains "2.2.5". The string was fixed in the 2.2.6.1 tarball (reading "2.2.6.1").
